src/TestEnvironment.Docker.Containers.Elasticsearch/TestEnvironment.Docker.Containers.Elasticsearch.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Add to README some description of these new package version changes and add workaround with usage of https://www.nuget.org/packages/Microsoft.Bcl.AsyncInterfaces/ for applications which are running on .net core prior to 3.0 version

Reply:
@Hellevar Can you elaborate a bit more on what this workaround is?  I'm actually not familiar with it.  And I thought it was available on .netstandard 2.1 or .netcoreapp 3.0

Reply:
@jzabroski IAsyncDisposable was added to netstandard2.1, so it isn't available in previous versions directly. But with help of mentioned package - it is possible to use IAsyncDisposable in applications running on netstandard prior to 2.1 due to Duck Typing of IAsyncDisposable interface. As far as @Deffiss reverted target framework and added this package as dependency, conversation might be resolved

Reply:
Thank you. Helpful